src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -366,6 +366,17 @@ export default {
       delete extra_obj.assetsSummary.schemaKey;
       delete extra_obj.assetsSummary.numberOfBytes;
       delete extra_obj.assetsSummary.numberOfFiles;
+      _.map(extra_obj, (val) => {
+        if (Array.isArray(val) && val.length !== 0) {
+          _.map(val, (each_val) => {
```

Comment:
These two `_.map()` calls should be `_.forEach()` instead.

Even better would be to use the JavaScript native `Array.forEach()` function rather than lodash.

Reply:
the outer `_.map()` is on an object, not an array.

Reply:
for the outer one i changed it to `_.forEach` and the inner one to `Array.forEach`